sdk/python/kubeflow/storage_init_container/hugging_face.py

```diff
@@ -45,6 +45,8 @@ def load_config(self, serialised_args):
     def download_model_and_tokenizer(self):
         # implementation for downloading the model
         print("downloading model")
+        import transformers
```

Comment:
Why do you need to move this import under `download_model_and_tokenizer` ?

Reply:
I added them to remove dependency of kubeflow on transformer library but now I have changed it